core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -104,12 +94,23 @@ object TestUtils extends Logging {
    * Create a temporary relative directory
    */
   def tempRelativeDir(parent: String): File = {
-    val f = new File(parent, "kafka-" + random.nextInt(1000000))
-    f.mkdirs()
+    new File(parent).mkdirs()
```

Comment:
hey @ismael, any particular reason for not using `Files#createTempDirectory` here?